README.md

```diff
@@ -32,3 +33,24 @@ closely, some implement the same thing. A build and test waterfall allows us to:
   component.
 
 We should keep process to a minimum, try things out, see what works.
+
+# How do I run it?
+
+1. Get the sources: `$ git clone https://github.com/WebAssembly/waterfall.git`
+2. Run build.py `python src/build.py`
+
+Build.py has 3 types of actions: downloading/updating sources for tools and engines (sync), 
+building those sources (build), and running tests against them (test). Each of these types
+has multiple steps (e.g. a build step for each component).
+If you run build.py with no arguments, it will run all the sync, build, and test steps. If
+you make a change and only want to run a subset of steps, you can apply filters from the
+command line, via exclusions (to prevent specified steps from running) or inclusions
+(to run only the specified steps). Sync, build, and test exclusions are specified separately.
+For example:
+1. Do not sync any sources, build everything except LLVM, and run all tests:
+  `$ src/build.py --no-sync --build-exclude=llvm`
+2. Sync only WABT, build WABT and Binaryen, run everything other than the emscripten testsuites: 
+  `$ src/build.py --sync-include=wabt --build-include=wabt,binaryen --test-exclude=emtest,emtest-asm`
+
+The script should throw an error if you specify nonexistent steps or if you specify both includes and excludes for the
```

Comment:
Maybe add a note about how is possible to work within the waterfall checkout, and it uses its own branch names to avoid clobbering your work?

Maybe mention that everything the script does is written to `work`.. and that you can blow away just `work` if you want a completely fresh start?

Reply:
Done